composer.json

```diff
@@ -19,7 +19,7 @@
 	},
 	"require": {
 		"luyadev/luya-core" : "^1.0",
-		"luyadev/luya-module-cms": "^1.0 | ^2.0",
+		"luyadev/luya-module-cms": "*",
```

Comment:
> Allow any LUYA **admin** version

but you made a change for cms module?

How about breaking changes? Setting a `*` might result in composer installing a combination that is incompatible.

Reply:
sure, this works as well. We just have to send a PR on next release. If there is a BC break i would have to send a PR with adjustments anyhow, so the * would be fine i thought.